packages/backend-data/src/factory.ts

```diff
@@ -239,6 +239,7 @@ class DataGenerator implements ConstructContainerEntryGenerator {
         authorizationModes,
         outputStorageStrategy: this.outputStorageStrategy,
         functionNameMap,
+        stackMappings: this.props.experimentalStackMapping,
```

Comment:
Why is this `experimental` if it's mapping to construct property that is not experimental in underlying construct ?